documentation/modules/file-system-overhead.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: CONCEPT
+[id="file-system-overhead_{context}"]
+= File system overhead
+ifdef::rhv[]
+When you run a cold migration from {rhv-full} to the {ocp} cluster that {project-full} is deployed on, the migration allocates persistent volumes without Customized Data Importer (CDI). In these cases, you might need to adjust the file system overhead.
+endif::[]
+
+
+ifdef::ostack[]
+When you migrate from {osp} to the {ocp} cluster that {project-full} is deployed on, the migration allocates persistent volumes without  Customized Data Importer (CDI). In these cases, you might need to adjust the file system overhead.
+endif::[]
+
+If the configured file system overhead, which has a default value of 10%, is too low, the disk transfer will fail due to lack of space. In such a case, you would want to increase the file system overhead.
```

Comment:
The configured file system overhead has a default value of 10%. If that is too low, the disk transfer fails due to lack of space. In this case, increase the file system overhead.